src/main/java/main.java

```diff
@@ -0,0 +1,8 @@
+import com.epam.tat.module4.Calculator;
+
+public class main {
```

Comment:
Class name starts with Capital letter
```suggestion
public class Main {
```